### ISSUE TITLE
Add support for APKBUILD (Alpine Linux)

### ISSUE THREAD

**Issue by bartekpacia** [goreleaser/goreleaser#4708]
### Is your feature request related to a problem? Please describe.

I'd like GoReleaser to have explicit support for apk's APKBUILD format, used by Alpine Linux.

APKBUILD is very similar to Arch's PKGBUILD. The only difference (that I'm aware of) is that APKBUILD scripts must be pure `sh`, while PKGBUILD are `bash`.

### Describe the solution you'd like

It would be basically the same as the current `aurs` feature, but with a different name.

### Describe alternatives you've considered

Use `aurs` to push to alpine linux repos and somehow automate renaming PKGBUILD to APKBUILD?

### Search

- [X] I did search for other open and closed issues before opening this

### Supporter

- [ ] I am a [sponsor](https://github.com/sponsors/caarlos0/) or a [Pro](https://goreleaser.com/pro) customer

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by caarlos0** [comment]
nice, yeah, we can probably do it.

do you have any references you would like us to look at?

**Comment by bartekpacia** [comment]
Sure!

- [Official APKBUILD Reference](https://wiki.alpinelinux.org/wiki/APKBUILD_Reference)
- [Creating an Alpine package](https://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package)

Thank you for being a fantastic open-source maintainer.

**Comment by bartekpacia** [comment]
Hmm, I see [GoReleaser already has support for nFPM](https://goreleaser.com/customization/nfpm), and it says "GoReleaser can be wired to `nfpm` to generate and publish `.deb`, `.rpm`, `.apk`, and ArchLinux packages." Therefore I think there's at least some overlap between the feature I propose in this issue and the already-supported nFPM.

But then, GoReleaser has explicit support for [AURs](https://goreleaser.com/customization/aur/)... hmm.

---

[nFPM](https://goreleaser.com/customization/nfpm/) is in the "Packaging and Archiving" docs section, and [AUR](https://goreleaser.com/customization/aur/) is in the "Publish" docs section, so I understand that these 2 are different things, on different stages of build/deploy process.

**Comment by caarlos0** [comment]
yeah, nfpm generates the packages themselves, and goreleaser can use nfpm to do the same.

goreleaser also supports creating and pushing AUR PKGBUILD & co files.

I haven't looked into the docs of APKBUILD yet to decide if it makes sense to add it as well or not

**Comment by space-aditya** [comment]
I tried in #6626
